src/neo/SmartContract/ApplicationEngine.Binary.cs

```diff
@@ -23,6 +27,21 @@ protected internal StackItem BinaryDeserialize(byte[] data)
             return BinarySerializer.Deserialize(data, Limits.MaxStackSize, Limits.MaxItemSize, ReferenceCounter);
         }
 
+        protected internal string Itoa(BigInteger value)
+        {
+            return value.ToString();
+        }
+
+        protected internal BigInteger Atoi(string value)
+        {
+            if (!BigInteger.TryParse(value, out var ret))
```

Comment:
Check the size first?

Reply:
I thought it, but it will be checked during the conversion https://github.com/neo-project/neo-vm/blob/a14b934f5b5c8779164e3b96b276984b3a57ccd4/src/neo-vm/Types/Integer.cs#L29